### ISSUE TITLE
[CAIA a11y Research]: MHV Secure Messaging Supporting Rescheduled AT Sessions with Screen Readers (September 2023)

### ISSUE THREAD

**Issue by rbsingleton** [department-of-veterans-affairs/va.gov-team#65457]
## CAIA a11y Assignee(s)
- [x] Sara (@sara-amanda)
- [x] SK (@SarahKay8)
- [x]  Jamie (@coforma-jamie)

### About the Secure Messaging/MHV on Va.gov-ByLight Team

<details><summary>Toggle to view team information.</summary>

- Team name: Secure Messaging/ MHV on VA.gov - ByLight
- OCTO-DE product owner: Janie Tankard-Carnock
- Product manager: Bryan Riley
- Researchers: Ade Ajayi and Anne Costello 
- Accessibility specialist: Angela Fowler
- Content writer: 
- Slack channel: [#secure-messaging-ux-team](https://dsva.slack.com/archives/C05CD3ECB44)
</details>
 
## CAIA Support Request
- We had to reschedule moderated research sessions with screen reader users. 
- Our a11y specialist will be on vacation for the rescheduled sessions. 
- We request a11y support for these sessions. 

### Support Goal 
- Assist users with assistive technology, should any issues take place, during the sessions. 
- Identify and document any accessibility-related issues encountered in the user testing sessions.

### Deliverable: Assistive Technology User Research Sessions

> @adeajayii Thank you for including CAIA A11ys in your research sessions. 
> 
> Please find [CAIA's research artifact for the MHV Secure Messaging Research Team](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/Secure%20Messaging/2023-Sept-Oct-Secure-Messaging-accessibility-findings.md) in GitHub.
> 
> **_While CAIA cannot require you to complete any updates to your product, we can provide your team with recommendations to help facilitate a smooth process for your project._**
> 
> Should you have any questions, please let us know! We are happy to help!

[The above text was taken from this comment in this ticket. ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65457#issuecomment-1781166589)

## Original Request Details
<details><summary>Toggle to view original request information.</summary>

### Session Schedule (Tentative)
**Sessions are currently being scheduled but are tentatively planned for the following dates:** 
- **9/25:** 10am to 8pm ET
- **9/26 :** 10am to 8pm ET

**Additional times if needed:**
- **9/27:** 10am to 2pm ET
- **9/28:** 10am to 8pm

### Session Dates for Assignees
- [x] **`RCVD 9/25/2023 9 a.m.` session times for dates provided [via Slack](https://dsva.slack.com/archives/C01K37HRUAH/p1695646632166059) and [Canvas](https://dsva.slack.com/docs/T03FECE8V/F05TL0T3T42):** September 25 & 26, 2023 (back-up sessions Sept.27 and 28)
_There are currently three studies overlapping during this timeframe, and we will do our very best to provide support for all of your AT sessions, due to the high-priority of this task, as back-up support to Angela, who is out of the office for the rescheduled sessions._
- [x] Added date range to the [CAIA Capacity Calendar](https://calendar.google.com/calendar/embed?height=600&wkst=1&bgcolor=%23ffffff&ctz=America%2FNew_York&mode=WEEK&src=Y19mOTgyMzUwZmM3M2E4MTYyYTEwODhhYjE0ZDAwZGQzNzMzNWIyYjU1OWE2MGQ2YjY3ZWI4MDIxNDE2ZjgxYzJhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20&color=%23616161).

### Do you have an accessibility champion?

Yes, we have a champion

### If you have an accessibility champion, what's their name?

Angela Fowler - will be on vacation during these sessions

### Calendar checking

- [x] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [X] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [x] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

### When is the timeframe for your testing?

9/25 & 9/26 : 10am to 8pm ET Additional time if needed 9/27: 10am to 2pm ET 9/28: 10am to 8pm

</details>

## Supporting artifacts

<details><summary>Toggle to view the research plan and conversation guide for SM</summary>

- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/secure-messaging/research/2023-09-usability-round3-phase0/research-plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/secure-messaging/research/2023-09-usability-round3-phase0/conversation-guide.md)

</details>

## Next steps

<details><summary>Toggle to view the completed, next steps</summary>

- [x] Post a link to this ticket in the accessibility-help Slack channel and tag Terry Nichols.
- [x] Your team's researchers and observers read through platform guidance on research with assistive technology users
- [x] CAIA Specialists are `assigned` to your research, be sure to add the respective a11y specialist(s) to your session invites (include the Zoom or Teams link).
    - [x] **`RCVD 9/25/2023 9 a.m.` session times for dates provided [via Slack](https://dsva.slack.com/archives/C01K37HRUAH/p1695646632166059) and [Canvas](https://dsva.slack.com/docs/T03FECE8V/F05TL0T3T42):** September 25 & 26, 2023 (back-up sessions Sept.27 and 28)
_There are currently three studies overlapping during this timeframe, and we will do our very best to provide support for all of your AT sessions, due to the high-priority of this task, as back-up support to Angela, who is out of the office for the rescheduled sessions._
- [x] CAIA Specialists review your conversation guide and research plan
- [x] Provide specialists your participant details like skill level, operating system, assistive tech used, browser
- [x] CAIA specialists will provide you an optional feedback form to let us know how we can support y'all better into the future.
- [x] [CAIA Specialists create a research notes template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65457#issuecomment-1781166589)
- [x] CAIA Post research, specialists will add accessibility findings to your research folder

</details>

**Comment by sara-amanda** [comment]
## CAIA Update
- There are session schedule details available, just yet.
- @artsymartha68 provided an update in the a11y sync meeting today (9/21/2023) that scheduling efforts are ongoing to contact qualifying screen reader users as participants for this study.

**Comment by sara-amanda** [comment]
## CAIA Update
### Screen Reader Sessions

- **`RCVD 9/25/2023 9 a.m.` session times for dates provided [via Slack](https://dsva.slack.com/archives/C01K37HRUAH/p1695646632166059) and [Canvas](https://dsva.slack.com/docs/T03FECE8V/F05TL0T3T42):** September 25 & 26, 2023 (back-up sessions Sept.27 and 28) - 
- **Note**: _There are currently three studies overlapping during this timeframe, and we will do our very best to provide support for all of your AT sessions, due to the high-priority of this task, as back-up support to Angela, who is out of the office for the rescheduled sessions._

- These dates and times have been added to the CAIA Capacity Calendar as of 9:27 a.m. 9/25/2023. 
cc: @SarahKay8 @coforma-terry @coforma-jamie 


### Time Slots and CAIA A11y Availability
`Copied from Canvas in Slack as of 9:25 a.m. 9/25/2023`
 **[Canvas](https://dsva.slack.com/docs/T03FECE8V/F05TL0T3T42) should be regarded as the source of truth c/o Ade Ajayi** (GitHub username unkown at this time)
#### Monday
#1 Monday, September 25th: 10am – 11:30am 
Moderator: Ade Ajayi 
Notetaker: Emily Taylor 
A11Y Support: `Sara`
Zoom: https://us06web.zoom.us/j/87016607316 

#2 Monday, September 25th- 3:00pm – 3:30pm 
Moderator: Ade Ajayi 
Notetaker: Emily Taylor 
A11Y Support: `Sara`
Zoom: https://us06web.zoom.us/j/83408568111 

#3 Monday, September 25th- 5:00pm – 6:30pm 
Moderator: Ade Ajayi 
Notetaker: Emily Taylor 
A11Y Support: `TBD - asking CAIA team members due to past office hours and OOO for two team members`
Zoom: https://us06web.zoom.us/j/83880313305 

------------------------------------------------------------------------------------------------------
#### Tuesday
#4 Tuesday, September 26th- 10am – 11:30am 
Moderator: Ade Ajayi 
Notetaker: Emily Taylor 
A11Y Support:  `Sara`
Zoom: https://us06web.zoom.us/j/87602819113 


#5 Tuesday, September 26th- 2pm – 3:30pm 
Moderator: Ade Ajayi 
Notetaker: Emily Taylor 
A11Y Support: `TBD`
Zoom: https://us06web.zoom.us/j/88517917150 

------------------------------------------------------------------------------------------------------
#### Wednesday
#6 Wednesday, September 27th – 10am – 11:30am 
Moderator: Ade Ajayi 
Notetaker: Emily Taylor 
A11Y Support: `TBD`
Zoom: https://us06web.zoom.us/j/81255830419

**Comment by sara-amanda** [comment]
## CAIA Deliverable
### Assistive Technology Research Sessions

**_Thank you for including CAIA A11ys in your research sessions._** 

Please find [CAIA's research artifact for the MHV Secure Messaging Research Team](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/Secure%20Messaging/2023-Sept-Oct-Secure-Messaging-accessibility-findings.md) in GitHub. 

_While CAIA cannot require you to complete any updates to your product, we can provide your team with recommendations to help facilitate a smooth process for your project._

Should you have any questions, please let us know! We are happy to help! 

cc: @coforma-terry @SarahKay8 @coforma-jamie
